Assessment/CustomerCommand/Api/ProfileInterface.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Assessment\CustomerCommand\Api;
+
+interface ProfileInterface
+{
+    public function import($source);
```

Comment:
```suggestion
    /**
     * @param <data_type> $source
     * @return <data_type>
     * /
    public function import($source);
```
Please try to add PHP doc block for all of your methods.

Reply:
added PHP doc block for all methods